### ISSUE TITLE
Null Reference Exception, looking for Texture2D that has been destroyed. Error reoccurs in Unity Console

### ISSUE THREAD

**Issue by jessemcculloch** [github-beta/unity-preview#35]
Please fill out the data below and include a zip with the Unity log file and the extension log file.
On Windows, the extension log file is at `%LOCALAPPDATA%\GitHubUnityDebug\github-unity.log`
On macOS, the extension log file is at `~/.local/share/GitHubUnityDebug/github-unity.log`
[github-unity.zip](https://github.com/github-beta/unity-preview/files/966277/github-unity.zip)

**Version of Unity**: 5.5.3f1
Version of GitHub for Unity (see console output): 0.9.1.0
**Platform (eg. Windows, macOS, Linux):** Windows

There is a reoccurring error showing up in the unity console.

MissingReferenceException: The object of type 'Texture2D' has been destroyed but you are still trying to access it.
Your script should either check if it is null or you should not destroy the object.
UnityEngine.Texture.get_width () (at C:/buildslave/unity/build/artifacts/generated/Metro/runtime/TextureBindings.gen.cs:46)
GitHub.Unity.ProjectWindowInterface.OnProjectWindowItemGUI (System.String guid, Rect itemRect) (at /Users/shana/code/work/github/UnityInternal/src/UnityExtension/Assets/Editor/GitHub.Unity/UI/ProjectWindowInterface.cs:225)
UnityEditor.ObjectListArea+LocalGroup.DrawItem (Rect position, UnityEditor.FilterResult filterItem, UnityEditor.BuiltinResource builtinResource, Boolean isFolderBrowsing) (at C:/buildslave/unity/build/Editor/Mono/ObjectListLocalGroup.cs:855)
UnityEditor.ObjectListArea+LocalGroup.DrawInternal (Int32 beginIndex, Int32 endIndex, Single yOffset) (at C:/buildslave/unity/build/Editor/Mono/ObjectListLocalGroup.cs:192)
UnityEditor.ObjectListArea+Group.Draw (Single yOffset, Vector2 scrollPos) (at C:/buildslave/unity/build/Editor/Mono/ObjectListGroup.cs:144)
UnityEditor.ObjectListArea.HandleListArea () (at C:/buildslave/unity/build/Editor/Mono/ObjectListArea.cs:1367)
UnityEditor.ObjectListArea.OnGUI (Rect position, Int32 keyboardControlID) (at C:/buildslave/unity/build/Editor/Mono/ObjectListArea.cs:562)
UnityEditor.ProjectBrowser.OnGUI () (at C:/buildslave/unity/build/Editor/Mono/ProjectBrowser.cs:1754)
System.Reflection.MonoMethod.Invoke (System.Object obj, BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) (at /Users/builduser/buildslave/mono/build/mcs/class/corlib/System.Reflection/MonoMethod.cs:222)

**Comment by StanleyGoldman** [comment]
Hey @jessemcculloch thanks for the bug report.
I've been tracking that bug on my end and should have a fix for it soon.

Just to be sure, could you try the latest release and see if it still occurs.
You will need to delete the GitHub folder from your project before starting unity, so it doesn't complain about files in use.

**Comment by StanleyGoldman** [comment]
I updated your original message with the version of the plugin you were using.

**Comment by StanleyGoldman** [comment]
So I read through the stack trace and compared code, this bug is a duplicate of #31.
It is fixed in the current release.
